backend/handlers/get_rule_graph.go

```diff
@@ -0,0 +1,69 @@
+package handlers
+
+import (
+	"github.com/Zeus-Labs/ZeusCloud/rules/types"
+	"log"
+	"net/http"
+
+	"github.com/Zeus-Labs/ZeusCloud/rules/attackpath"
+	"github.com/neo4j/neo4j-go-driver/v4/neo4j"
+)
+
+var ruleGraphMap = map[string]types.Rule{
```

Comment:
Don't use a separate `ruleGraphMap`. This makes it less maintainable as someone has to remember to update this piece of code when writing a new rule. People should write rules in their rule file and things should just work everywhere else, more or less.

We already have a mapping of the attack path rule struct to rule/uid. Just loop through `AttackPathsRulesToExecute` global variable and find the attack path that matches the rule uid that's passed in. Performance impact of looping through few hundreds strings is minimal.